charts/vault-csi-provider/values.yaml

```diff
@@ -6,11 +6,17 @@
 # image -- specifies the image to use for the secrets-store-csi-driver
 image:
   repository: hashicorp/secrets-store-csi-driver-provider-vault
-  tag: 0.0.6
+  tag: 0.0.7
   pullPolicy: IfNotPresent
 
 # resources -- Set the limits and requests on vault csi-driver-provider pod resources
-resources: {}
+resources:
+  requests:
```

Comment:
Why deviate from the default of leaving this empty? I can't find a mention of leaving it empty in the best-practices but an empty chart made with `helm create` has this info:
>   We usually recommend not to specify default resources and to leave this as a conscious
   choice for the user. This also increases chances charts run on environments with little
   resources, such as Minikube. If you do want to specify resources, uncomment the following
   lines, adjust them as necessary, and remove the curly braces after 'resources:'.

IMO leaving this as a conscious choise to the user still holds true.

Reply:
It's all taken from the default deployment/daemonset.yaml of the upstream repo. https://github.com/hashicorp/secrets-store-csi-driver-provider-vault/blob/master/deployment/provider-vault-installer.yaml

Reply:
Would you suggest discarding their upstream input and just leaving this blank?

Reply:
if some resources are required that the pod can work then it makes sense to define the minimal `requests`. `limits` are not needed IMHO

Reply:
According to theyr upstream chart which will be released soon, they also set the resources to nil. I will look into that later this month or in April so we can depreacate mantaining `vault-auth` and `vault-csi-provider` ourselfes.

Reply:
Removed and set to nil